docs/cudf/source/user_guide/performance_comparisons.ipynb

```diff
@@ -0,0 +1,1699 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    }, index=["to_parquet", "read_parquet", "to_csv", "read_csv", "groupby", "merge"]).astype('int')
</pre><p>Don't cast this as int. These are floating values and truncation is inaccurate if the speedups are ~1 for any set of parameters.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/12595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
(If you are doing this cast to get more human-readable numeric labels for your plots, use a formatter instead of truncating the values in the dataframe.)

Reply:
Done